src/App/Services/Tamara_Client.php

```diff
@@ -126,7 +127,7 @@ function ( &$tmp_item, $tmp_index ) {
 						$tmp_item = $tmp_item['error_code'] ?? null;
```

Comment:
We may need to put `General_Helper::convert_message` for `$tmp_item['error_code']`